packages/ra-core/src/actions/accumulateActions.ts

```diff
@@ -2,7 +2,19 @@ import { crudGetMany, crudGetMatching } from './dataActions';
 
 export const CRUD_GET_MANY_ACCUMULATE = 'RA/CRUD_GET_MANY_ACCUMULATE';
 
-export const crudGetManyAccumulate = (resource, ids) => ({
+export const crudGetManyAccumulate = (
+    resource: string,
+    ids: []
+): {
+    type: string;
```

Comment:
can't you be stricter with the type, given that we always know *which* string the function returns?